### PR TITLE
Fix Bug 1399276, post launch content update for dev whatsnew

### PR DESCRIPTION
**Pull request by schalkneethling**
## Description
Post launch copy updates for developer whatsnew page

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1399276

## Testing
Ensure no regressions
Ensure all new strings are correctly wrapped in conditionals and for l10n

### CODE REVIEW

**Comment by peiying2** [comment]
This page contains the same l10n content as PR5202, it seems. Could you put test URL here too?

File: bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html [hunk 1]
```diff
@@ -68,13 +68,23 @@ <h2 class="section-title"><span>{{ _('New Tools') }}</span></h2>
         <div class="section-body">
           <h3 id="devtools" class="body-title">{{ _('Firefox DevTools') }}</h3>
 
-          <img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-debugger-ani.gif') }}" alt="">
+          <a href="http://firefox-dev.tools/" rel="external"><img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-debugger-ani.gif') }}" {% if l10n_has_tag('quantum-firstrun-whatsnew') %}alt="{{ _('Learn more about Firefox developer tools') }}"{% endif %}></a>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html]: This should fall back to an empty alt for locales without the tag.

File: bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html [hunk 2]
```diff
@@ -84,7 +94,8 @@ <h2 class="section-title"><span>{{ _('Innovative Features') }}</span></h2>
         <div class="section-body">
           <h3 id="cssgrid" class="body-title">{{ _('Master CSS Grid') }}</h3>
 
-          <img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-cssgrid-ani.gif') }}" alt="">
+          <a href="https://mozilladevelopers.github.io/playground/" rel="external"><img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-cssgrid-ani.gif') }}" {% if l10n_has_tag('quantum-firstrun-whatsnew') %}alt="{{ _('Learn more about CSS Grid') }}"{% endif %}></a>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html]: Fall back to empty alt.

File: bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html [hunk 3]
```diff
@@ -126,73 +137,133 @@ <h3 class="body-title">{{ _('Build and Perfect your sites<br> with Firefox DevTo
           <li class="gallery-item feature">
             <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-inspector.svg') }}" alt="">
             <h3 class="name">{{ _('Inspector') }}</h3>
-            <p class="desc">{{ _('Inspect and refine code to build pixel-perfect layouts.') }}</p>
+            <p class="desc">
+              {{ _('Inspect and refine code to build pixel-perfect layouts.') }}
+              {% if l10n_has_tag('quantum-firstrun-whatsnew') %}
+                <p class="features-learn-more"><a href="https://developer.mozilla.org/docs/Tools/Page_Inspector" rel="external">{{ _('Learn more about Page Inspector') }}</a></p>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html]: Nested paragraph here and elsewhere.

File: bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html [hunk 4]
```diff
@@ -126,73 +137,133 @@ <h3 class="body-title">{{ _('Build and Perfect your sites<br> with Firefox DevTo
...
           </li>
 
           <li class="gallery-item feature">
             <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-memory.svg') }}" alt="">
             <h3 class="name">{{ _('Memory') }}</h3>
-            <p class="desc">{{ _('Find memory leaks and make your application zippy.') }}</p>
+            <p class="desc">
+              {{ _('Find memory leaks and make your application zippy.') }}
+              {% if l10n_has_tag('quantum-firstrun-whatsnew') %}
+                <p class="features-learn-more"><a href="https://developer.mozilla.org/en-US/docs/Tools/Memory" rel="external">{{ _('Learn more about Memory Tools') }}</a></p>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html]: Remove `en-US`

File: bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html [hunk 5]
```diff
@@ -126,73 +137,133 @@ <h3 class="body-title">{{ _('Build and Perfect your sites<br> with Firefox DevTo
...
           </li>
 
           <li class="gallery-item feature">
             <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-scratchpad.svg') }}" alt="">
             <h3 class="name">{{ _('Scratchpad') }}</h3>
-            <p class="desc">{{ _('Edit, write, run and execute JavaScript in real time.') }}</p>
+            <p class="desc">
+              {{ _('Edit, write, run and execute JavaScript in real time.') }}
+              {% if l10n_has_tag('quantum-firstrun-whatsnew') %}
+                <p class="features-learn-more"><a href="https://developer.mozilla.org/en-US/docs/Tools/Scratchpad" rel="external">{{ _('Learn more about Scratchpad') }}</a></p>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html]: Remove `en-US`

**Comment by schalkneethling** [comment]
Thanks for the review @craigcook ~ updated

**Pull request merged**